### ISSUE TITLE
vue-i18n throws many warnings

### ISSUE THREAD

**Issue by gplusdotgr** [vuejs/vue-test-utils#639]
### Version
1.0.0-beta.16

### Reproduction link
[https://vue-test-utils.vuejs.org/](https://vue-test-utils.vuejs.org/)

### Steps to reproduce
I am using vue-i18n in my application, loading it via my Vuex store. Vue-test-utils throws a warning for every string:
```
WARN: '[vue-i18n] Cannot translate the value of keypath "error_went_wrong". Use the value of keypath as default'
WARN: '[vue-i18n] Cannot translate the value of keypath "common_retry". Use the value of keypath as default'
WARN: '[vue-i18n] Cannot translate the value of keypath "error_notification_history". Use the value of keypath as default'
```

### What is expected?
No warnings should be throw

### What is actually happening?
Warnings are thrown

**Comment by LinusBorg** [comment]
> I am using vue-i18n in my application, loading it via my Vuex store.

Please provide a reproduction. We can't work with a one-sentence description.

**Comment by LinusBorg** [comment]
Closing for inactivity and missing reproduction.

We can consider reopening once a reproduction is provided.

**Comment by gplusdotgr** [comment]
I think its practically impossible to reproduce this as its in a complex corporate private repo with many dependencies but I understand that this could be the way to debug it.

I googled around and I've seen several issues with the same warning so I imagined there's some past experience/helpful tips on why this is happening.

**Comment by LinusBorg** [comment]
> past experience/helpful tips 

Consider asking on forum.vuejs.org or chat.vuejs.org

> why this is happening.

Since you didn't share your test code, I have to blindly guess what you might have done wrong. I will guess once:

You did not coreectly register the plugin with the local Vue instance created with `createLocalVue()`, or did not pass the instance of i18n to the component, or something like that.

If you did take care of that, then I'm out of things to guess.

**Comment by gplusdotgr** [comment]
good guess as I didn't actually do that. Do I have to do it in every component using it or can I do it somehow in my setup?

**Comment by LinusBorg** [comment]
I'm not very familiar with th different ways of setting up that pugin - a good moment to switch to one of the sources I mentioned to ask other for help with that.

**Comment by eddyerburgh** [comment]
Likely this is due to how we've implemented sync behavior in beta.12. There are similar issues about warnings being thrown because we set the main render watcher to sync after it's initially run.

The temporary fix is to run with `sync` set to false, and use Vue.nextTick to await re renders:

```js
test('use Vue.nextTick', (done) => {
  const wrapper = mount(TestComponent, { sync: false })
  wrapper.trigger('click')
  Vue.nextTick(() => {
    expect(wrapper.text()).toBe('updated')
    done()
  })
})
```
We're working on a solution right now that should be out in the next few weeks.
